client/src/tests/components/SignUp.spec.js

```diff
@@ -0,0 +1,40 @@
+import expect from 'expect';
+import React from 'react';
+import { shallow } from 'enzyme';
+import { SignUpForm } from '../../components/signup/SignUp';
+
+/**
+ * Wrapper function for the shallow function
+ *  @returns {*} a component
+ */
+function setup() {
```

Comment:
**Nitpick**
You don't need this function. Simply pass in the props to the component directly